File: backend/job_trackr/apps/jobs/candidacies/choices.py
# SPDX-License-Identifier: AGPL-3.0-or-later
# File: backend/job_trackr/apps/jobs/candidacies/choices.py

from django.db import models


class CandidacyStatus(models.TextChoices):
    APPLIED = "applied", "Applied"
    INTERVIEW = "interview", "Interview"
    TECHNICAL_TEST = "technical_test", "Technical test"
    OFFER = "offer", "Offer"
    REJECTED = "rejected", "Rejected"
    WITHDRAWN = "withdrawn", "Withdrawn"

File: frontend/src/features/jobs/candidacies/choices.ts
// SPDX-License-Identifier: AGPL-3.0-or-later
// File: frontend/src/features/jobs/candidacies/choices.ts

export const statusChoices = [
  { value: "applied", label: "Applied" },
  { value: "interview", label: "Interview" },
  { value: "technical_test", label: "Technical test" },
  { value: "offer", label: "Offer" },
  { value: "rejected", label: "Rejected" },
  { value: "withdrawn", label: "Withdrawn" },
] as const;

export type CandidacyStatus = (typeof statusChoices)[number]["value"];
